plugins/UM3NetworkPrinting/resources/qml/PrinterCard.qml

```diff
@@ -118,7 +118,10 @@ Item {
                     Item {
                         id: machineNameLabel;
                         height: UM.Theme.getSize("monitor_text_line").height;
-                        width: Math.round(parent.width * 0.3);
+                        width: {
```

Comment:
Couldn't this just be `width: Math.round(parent.width * (printer ? 0.75 : 0.3)) // width depends on actual printer vs skeleton`?

Reply:
Yes, although I decided at the last second it was a little bit hard to read at a glance and figured "why not expand it slightly".

I'll change it if you think single line is better.